documentation/bolt_running_plans.md

```diff
@@ -16,12 +16,20 @@ For example, imagine a plan that deploys a load balancer. The plan is located at
 target that the plan runs its tasks or functions on. If your load balancer was
 `lb.myorg.com`, you would use the following command to run the plan:
 
-```
-bolt plan run mymodule::myplan load_balancer=lb.myorg.com
-```
+- _\*nix shell command_
+
+  ```shell
+  bolt plan run mymodule::myplan load_balancer=lb.myorg.com
+  ```
+
+- _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltPlan -Name mymodule::myplan -Params load_balancer=lb.myorg.com
```

Comment:
Do you actually need `-Params` here? That seems to be inconsistent between a few of the examples.

Reply:
Ah, ya, that shouldn't be there. The examples originally showed passing the params as objects, but I decided to keep it similar to the shell commands.